community-operators/okd-windows-machine-config-operator/0.0.1/okd-windows-machine-config-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,229 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/okd-windows-machine-config-operator:4.6-1d646e8
+    createdAt: 2020-09-01T11:59:59Z
+    description: An operator which allows for Windows workloads to be run within OKD
```

Comment:
`An operator that enables Windows workloads on OKD`

Reply:
An operator that enables Windows container workloads on OKD